frontend/src/features/properties/map/research/add/ResearchProperties.tsx

```diff
@@ -3,6 +3,7 @@ import { StyledFormSection } from 'features/mapSideBar/tabs/SectionStyles';
 import MapSelectorContainer from 'features/properties/selector/MapSelectorContainer';
 import { IMapProperty } from 'features/properties/selector/models';
 import { FieldArray, useFormikContext } from 'formik';
+import { isEqual } from 'lodash';
```

Comment:
nit: it's usually better to import from "lodash/isEqual" to help with tree shaking but no big deal to leave like that